packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -133,7 +133,7 @@ export function useSliderThumb(
 
   let onUp = (e) => {
     let id = e.pointerId ?? e.changedTouches?.[0].identifier;
-    if (id === currentPointer.current) {
+    if (id == currentPointer.current) {
```

Comment:
2nd fix: instead of moving the ColorSlider thumb into ColorSlider track element, this fixes it so mouseup properly calls onUp since the comparision between `id` and `currentPointer.current` is `undefined` vs `null`. 

https://github.com/adobe/react-spectrum/commit/055dd9c3b99822f6ef87037c3724f5bba855fd98 was the change that introduced the `currentPointer` stuff and indicated that it was to fix multitouch when dragging on ColorWheel/Slider, the above fix doesn't seem to cause a regression